src/rateEngine/BillingDeterminantFactory.ts

```diff
@@ -11,25 +11,28 @@ import SurchargeAsPercent, { SurchargeAsPercentArgs } from './billingDeterminant
 import HourlyEnergy, { HourlyEnergyArgs } from './billingDeterminants/HourlyEnergy';
 import DemandTiersInMonths from './billingDeterminants/DemandTiersInMonths';
 import DemandTimeOfUse, { DemandTimeOfUseArgs } from './billingDeterminants/DemandTimeOfUse';
+import DemandPerDay, { DemandPerDayArgs } from './billingDeterminants/DemandPerDay';
 
 export type RateElementType =
-  'EnergyTimeOfUse' |
-  'BlockedTiersInDays' |
-  'BlockedTiersInMonths' |
-  'FixedPerDay' |
-  'FixedPerMonth' |
-  'MonthlyEnergy' |
-  'MonthlyDemand' |
-  'SurchargeAsPercent' |
-  'HourlyEnergy' |
-  'DemandTimeOfUse' |
-  'DemandTiersInMonths';
+  | 'EnergyTimeOfUse'
```

Comment:
looks like we need to set up linting/prettier here! (def not necessary for this PR)